deploy/examples/kubernetes/travel-agency/deploy.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/env bash
+
+EXAMPLES_DIR=./deploy/examples
+TARGET_DIR="${EXAMPLES_DIR}/kubernetes/travel-agency"
+STRIMZI_VERSION=0.17.0
+if [[ -z ${PROJECT_NAME} ]]; then
+  PROJECT_NAME=kogito
+fi
+
+echo "Deploying the Travel Agency Demo on ${PROJECT_NAME} namespace"
+oc project "$PROJECT_NAME"
+
+echo "Installing Infinispan Operator"
+oc apply -f https://raw.githubusercontent.com/infinispan/infinispan-operator/1.1.1.Final/deploy/crd.yaml -n ${PROJECT_NAME}
```

Comment:
shouldn't use kubctl since it is a k8s example?

Reply:
Yeah, I can change that.